nexus/reconfigurator/execution/src/omicron_zones.rs

```diff
@@ -33,6 +34,16 @@ pub(crate) async fn deploy_zones(
             {
                 Some(sled) => sled,
                 None => {
+                    if config.zones.iter().all(|c| {
+                        c.disposition == BlueprintZoneDisposition::Expunged
```

Comment:
Do you think we'll do this frequently enough it warrants something like `config.are_all_zones_expunged()`?

Reply:
I don't think it will be used that frequently, but I like the method better. Added in 8b07fa885bdf94ed2009e1bda4da5d34dd6b4c72